src/main/commandServer/__tests__/httpCommandServer.spec.ts

```diff
@@ -0,0 +1,51 @@
+import os from 'os';
+import path from 'path';
+
+import { HttpCommandServer } from '../httpCommandServer';
+import { spawnFile } from '@/utils/childProcess';
+import resources from '@/utils/resources';
+
+function exeIt(name: string) {
+  return os.platform().startsWith('win') ? `${ name }.exe` : name;
+}
+
+describe(HttpCommandServer, () => {
```

Comment:
Running this as a `HttpCommandServer` test (rather than an `rdctl` test) seems odd, but 🤷

Reply:
There's no `rdctl.ts` to run a unit test against.

Reply:
Also, I added a `cmd.SilenceUsage = true` to `listSettings.go` on this branch, so it's weird that the output is showing the usage if you rebuilt it.

And, I was focused on the lima VM. There's no point running this test on Windows, although I see the output isn't that great if there's no `rancher-desktop` WSL on Windows:
```
There is no distribution with the supplied name.
Error: exit status 0xffffffff
```